datafusion-examples/examples/planner_api.rs

```diff
@@ -123,5 +124,19 @@ async fn to_physical_plan_step_by_step_demo(
             .plan
     );
 
+    // Call the physical optimizer with an existing physical plan (in this
```

Comment:
💯 
FYI @waynexia -- the example you added in https://github.com/apache/datafusion/pull/11716 is already being extended and used 🎉